src/themes/shared/breakpoints.ts

```diff
@@ -1,8 +1,33 @@
-export const breakpoints = {
+export const emBreakpoints = {
   base: '0em',
   xs: '23.4375em',
   sm: '30em',
   md: '48em',
   lg: '64em',
   xl: '80em',
 };
+
+type BreakpointName = keyof typeof emBreakpoints;
+
+export const pxBreakpoints = Object.fromEntries(
```

Comment:
I believe that `.fromEntries` takes a generic so that you don't need to cast it: 
`Object.fromEntries<Record<BreakpointName, string>>`

Reply:
It only takes a type argument of the `Value` and returns `{ [key: string]: Value }` so you're losing the type of the `key` which I'm trying to preserve.

Reply:
got it 👍